go-console/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["ms-vscode.cpptools", "golang.go"]
```

Comment:
to maintain consistency between templates it would be nice to put entry by line:
```json
{
  "recommendations": [
        "ms-vscode.cpptools", 
        "golang.go"
   ]
}
```